addons/medical_blood/XEH_postInit.sqf

```diff
@@ -39,7 +39,7 @@ if (isServer) then {
 
     ["CAManBase", "hit", {
         params ["_unit"];
-        if (GVAR(enabledFor) == 1 && {!isPlayer _unit || {_unit == ACE_player}}) exitWith {};
+        if (GVAR(enabledFor) == 1 && {_unit != ACE_player}) exitWith {};
```

Comment:
`isPlayer` check should remain. `isPlayer _unit` does not always equal `ACE_player`. Example; in cases of remote control.